### ISSUE TITLE
More human readable IBAN

### ISSUE THREAD

**Issue by J0000** [Project60/org.project60.sepa#423]
On different places where the IBAN occurs, it would be nice to see it formatted in the official way ( XX00 0000 0000 0000 0000 00). It's easier for the user to control the correctness with blanks on the right places.

Also in the dropdown-combo which contains the IBAN it would be nice to see the number formatted. Perhaps the IBAN could be entered with whitespaces which can easily removed afterwards, too.

**Comment by systopia** [comment]
I agree, that would be nice. Is the format you mentioned (blocks of four) applicable for all IBANs or is that country specific?

**Comment by J0000** [comment]
In Germany it's defined in the normation DIN 5008 as described. The official IBAN standard doesn't have any ISO normination, but in all countries the used format should be the same: blocks of four (the length differs) and the length of last block depends on the total length

https://en.wikipedia.org/wiki/International_Bank_Account_Number
